fvm/src/call_manager/default.rs

```diff
@@ -153,6 +160,22 @@ where
         self.num_actors_created += 1;
         ret
     }
+
+    fn get_fuel_consumed_snapshot(&self) -> u64 {
```

Comment:
You should be able to store these on the `InvocationContext` itself. We _cannot_ set them on the CallManager as sub-calls will clobber the values stored by parents.

Reply:
It also means you can avoid modifying the kernel and call manager.

Reply:
Ah, yes, we discussed then in person, but it slipped my mind. Thanks!